apps/meteor/app/lib/server/functions/saveCustomFieldsWithoutValidation.ts

```diff
@@ -48,10 +60,12 @@ export const saveCustomFieldsWithoutValidation = async function (userId: string,
 		}
 	});
 
-	await Users.updateFromUpdater({ _id: userId }, updater);
+	if (!_updater) {
```

Comment:
Since you are providing `updater` to the model call, is there a necessity to do this check?

Reply:
This is a standalone method that is called in other places (users.create for example). This check avoids the update if the updater comes from outside the function, because in that case the caller is responsible for commiting the updates